assets/js/base/components/sidebar-layout/sidebar-layout.js

```diff
@@ -3,27 +3,20 @@
  */
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
-import { useContainerQueries } from '@woocommerce/base-hooks';
+import { ContainerQueryContextProvider } from '@woocommerce/base-context';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 
 const SidebarLayout = ( { children, className } ) => {
-	const [ resizeListener, containerQueryClassName ] = useContainerQueries();
-
 	return (
-		<div
-			className={ classNames(
-				'wc-block-sidebar-layout',
-				className,
-				containerQueryClassName
-			) }
+		<ContainerQueryContextProvider
```

Comment:
This is mostly a question to @Aljullu that I forget to mention in the initial PR, why is this wrapping the sidebar only? wouldn't we need it in the main view?

Reply:
Actually it's wrapping everything, the structure is like this:
```HTML
<div class="wc-block-sidebar-layout">
	<div class="wc-block-main">...</div>
	<div class="wc-block-sidebar">...</div>
</div>
```
I know the class name `wc-block-sidebar-layout` for the wrapper might be a bit misleading, but couldn't find a better alternative. I would be happy to rename it if somebody has a better idea. :slightly_smiling_face: